modules/swagger-codegen/src/main/resources/python/rest.mustache

```diff
@@ -76,10 +76,33 @@ class RESTClientObject(object):
         # key file
         key_file = Configuration().key_file
 
+       # proxy
+        proxy = Configuration().proxy
+
+        # https pool manager
+        if proxy:
+            self.pool_manager = urllib3.ProxyManager(
+                num_pools=pools_size,
+                maxsize=maxsize,
+                cert_reqs=cert_reqs,
+                ca_certs=ca_certs,
+                cert_file=cert_file,
+                key_file=key_file,
+                proxy_url=proxy
+            )
+        else:
+            self.pool_manager = urllib3.PoolManager(
+                num_pools=pools_size,
+                maxsize=maxsize,
+                cert_reqs=cert_reqs,
+                ca_certs=ca_certs,
+                cert_file=cert_file,
+                key_file=key_file
+            )
+
         # https pool manager
         self.pool_manager = urllib3.PoolManager(
```

Comment:
@krmaxwell Should this be removed as the code you added above already takes care of `pool_manager` initialization.